dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/jaxrs/provider/ProviderFactory.java

```diff
@@ -283,7 +283,7 @@ public ClassLoader run() {
         return c;
     }
 
-    public static Object createJsonBindingProvider() {
+    public static Object createJsonBindingProvider(Iterable<ProviderInfo<ContextResolver<?>>> contextResolvers) {
```

Comment:
I see you add this contextResolvers parameter, but I don't see it being used in this method.

Reply:
We don't need it for Jackson in Jaxrs-2.0, hence it's not referenced. But the method API is shared between Jaxrs-2.0 and Jaxrs-2.1 so it has to be here.

Reply:
works for me... thanks